module/Photo/src/Photo/Model/Photo.php

```diff
@@ -319,6 +319,14 @@ public function getTags()
         return $this->tags;
     }
 
+    /**
+     * @return \Photo\Model\Hit
+     */
+    public function getHits()
```

Comment:
If you're only interested in the hitcount you can turn this in to `getHitCount()` and return `$this->hits->count()`. If I recall correctly that is a lot faster than retrieving all hits.